packages/react-native-worklets/Common/cpp/worklets/SharedItems/Shareables.cpp

```diff
@@ -187,6 +187,15 @@ jsi::Value makeShareableHostObject(
   return ShareableJSRef::newHostObject(rt, shareable);
 }
 
+jsi::Value makeShareableTurboModuleLike(
+    jsi::Runtime &rt,
+    const jsi::Object &object,
+    const jsi::Object &proto) {
```

Comment:
Can we already have a HO proto in the signature?

```suggestion
jsi::Value makeShareableTurboModuleLike(
    jsi::Runtime &rt,
    const jsi::Object &object,
    const jsi::HostObject &proto) {
```